### PR TITLE
[DOCS] Add PYTHON option to `kyuubi.operation.language` config

### PR DESCRIPTION
**Pull request by bowenliang123**
### _Why are the changes needed?_

- as pyspark is supported in #3780, exposing PYTHON option for `kyuubi.operation.language` as an experimental feature 

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by bowenliang123** [comment]
cc @cfmcgrady @pan3793

File: kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala [hunk 1]
```diff
@@ -2160,8 +2160,11 @@ object KyuubiConf {
   val OPERATION_LANGUAGE: ConfigEntry[String] =
     buildConf("kyuubi.operation.language")
       .doc("Choose a programing language for the following inputs" +
-        " <ul><li>SQL: (Default) Run all following statements as SQL queries.</li>" +
-        " <li>SCALA: Run all following input a scala codes</li></ul>")
+        "<ul>" +
+        " <li>SQL: (Default) Run all following statements as SQL queries.</li>" +
+        " <li>SCALA: Run all following input as scala codes</li>" +
+        " <li>PYTHON: (Experimental) Run all following input as PySpark codes</li>" +
```
Comment (bowenliang123) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala:2166]: which is better `as PySpark cods`, `as Python codes` or `as Python codes with Spark engine`?

**Review by pan3793 — APPROVED** [review]

File: kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala (hunk 1 above)
Comment (cfmcgrady) [reply kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala:2166]: +1 for `as Python codes with Spark engine`

Comment (bowenliang123) [reply kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala]: Good, updated.

**Review by cfmcgrady — APPROVED** [review]

**Comment by bowenliang123** [comment]
Thanks, merged to master/1.7 .

**Pull request closed without merging**